bandwidth-file-spec.txt

```diff
@@ -568,6 +569,32 @@
       The descriptor observed bandwidth mean for this relay in bytes per
       second.
 
+    "vote=" Bool SP
+
+      [Zero or one time.]
+
+      If the value is 0, Tor directory authorities MUST NOT vote on this relay.
+      This MAY be the case when this relay was not successfully measured but
+      it is reported in the Bandwidth File to diagnose why it was not measured.
+      If the value is 1 or the KeyValue is not present, Tor directory
+      authorities MUST vote on this relay.
+
+      Because Tor versions before 0.3.4 ignore this KeyValue, generator
```

Comment:
Insert the correct list of versions, which might still need updating if we have a new 0.4.0 alpha this week:
```suggestion
      Because Tor versions 0.3.4.11, 0.3.5.8, 0.4.0.2-alpha, and earlier
      ignore this KeyValue, generator implementations MUST set
```